src/dotnet/APIView/APIViewWeb/Repositories/PullRequestManager.cs

```diff
@@ -281,13 +287,8 @@ private ReviewModel CreateNewReview(PullRequestModel prModel)
                 var renderedCodeFile = new RenderedCodeFile(codeFile);
                 if (await IsReviewSame(review, renderedCodeFile))
                 {
-                    //Do not update the comment if review was already created and it matches with current revision.
-                    if (pullRequestModel.ReviewId != null)
-                        return "";
-
-                    //Baseline review was not created earlier or this is the first commit of PR
-                    stringBuilder.Append($"API changes are not detected in this pull request for `{codeFile.PackageName}`");
-                    return stringBuilder.ToString();
+                    // Disable the comment when API change is not detected.

```

Comment:
Do we want to disable the comment? My expectation is that we would just reuse the existing comment if it already exists and if not then we would add one saying there is no updates detected.

Reply:
We are already reusing existing comment. But this is still an issue when a PR touches multiple services it gets one comment per each package. We still don't have combined single comment for entire PR. It is one comment per package which can still be an issue. This PR is temporary solution until we build combined single comment taht includes link for each PR.